### ISSUE TITLE
[bug] Dragging/swiping on sliders with lots of items ignores "slideBy"

### ISSUE THREAD

**Issue by crystalenka** [ganlanyuan/tiny-slider#626]
__Issue description__:  I've played with various settings and the only consistent thing I can find is that the sliders that are buggy have lots of items.

Swiping on a touch device or dragging in the browser seems to drag the whole slider in free-for-all instead of moving the slider by one.

__Demo link/slider setting__:   https://dev.kp11studios.com/portfolio/shashank-and-supreeta
Working slider on same site with same (ish) settings: https://dev.kp11studios.com/portfolio/sam-axe-jester

_Tiny-slider version_:   2.9.3
_Browser name && version_:  All
_OS name && version_:   All

**Comment by ivgalic** [comment]
I'm experiencing the same issue, though I'm not sure if it's a bug or intended behaviour. Clicking 'next' or 'prev' goes through 'slide by' images, but swiping on mobile, or mouse dragging on desktop seems to slide through the whole carousel - images just flick through.

For me it seems to happen in all cases, even with few images (say 4).

**Comment by crystalenka** [comment]
I don't THINK it's intended behavior because prior versions of tinyslider did not do this... on phone it's certainly disconcerting, especially with 20+ images loaded.

**Comment by ivgalic** [comment]
You're right, the official demos don't seem to suffer from this either. It must be the structure of my/your html, but I can't see what I'm doing wrong.

**Comment by crystalenka** [comment]
The official demos have very few slides; I'm only experiencing it on sliders with 15+ full-width slides. :/ I don't even know how to begin debugging this.

**Comment by crystalenka** [comment]
@ganlanyuan Do you think you can help with this when you have time?

**Comment by Nico-Schaefer-2111** [comment]
@crystalenka @ivgalic  Did you find a solution to this? 

Kind regards

**Comment by ivgalic** [comment]
I'm sorry, but this was so long ago I can't remember even what the issue was... Probably better to find something that's still maintained?
